input/postgres/vacuum_progress.go

```diff
@@ -13,7 +13,18 @@ SELECT (EXTRACT(epoch FROM a.query_start)::int::text || pg_catalog.to_char(pid,
 			 (EXTRACT(epoch FROM COALESCE(backend_start, pg_catalog.pg_postmaster_start_time()))::int::text || pg_catalog.to_char(pid, 'FM0000000'))::bigint AS backend_identity,
 			 a.datname,
 			 (SELECT pg_catalog.regexp_matches(query, 'autovacuum: VACUUM (ANALYZE )?([^\.]+).([^\(]+)( \(to prevent wraparound\))?'))[2] AS nspname,
-			 (SELECT pg_catalog.regexp_matches(query, 'autovacuum: VACUUM (ANALYZE )?([^\.]+).([^\(]+)( \(to prevent wraparound\))?'))[3] AS relname,
```

Comment:
Is it intentional that this does *not* coalesce `nspname` like we do [here](https://github.com/pganalyze/collector/pull/62/files#diff-1f9a634167b0b4f0ff14633f304b9fb6R46)?

Reply:
Ah yes, because we don't have that table.